src/App.js

```diff
@@ -36,22 +36,24 @@ class App extends Component {
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
 
-    console.log(this.state);
+    // console.log(this.state);
   }
 
   onItemClicked = () => {
-    // Fill this in for Wave 3!
+    this.setState({points: this.state.points + 1});
   }
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
-               height={item.height}     // Height - used for a CSS style to position on the screen
-               layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
-               key={item.id}            // Key - to help React with performance
-
-               // Additional props (event callbacks, etc.) can be passed here
-             />;
+              height={item.height}     // Height - used for a CSS style to position on the screen
+              layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
+              key={item.id}            // Key - to help React with performance
+              
+              // Additional props (event callbacks, etc.) can be passed here
+              type={item.type}    // rock/litter/flowers/mushroom/etc
+              parentCB={this.onItemClicked}
```

Comment:
For clarity it would be best practice to name parentCB something like `addPointCallback`